### PR TITLE
WebRTC Stats Stream

### PR DESCRIPTION
**Pull request by ianwremmel**
This started out as "make a sample demonstrating mute and unmute", but ended up turning into "provide the hooks necessary to prove audio and video are flowing without a human being involved in the test".

### CODE REVIEW

File: packages/node_modules/@ciscospark/plugin-phone/src/call.js [hunk 1]
```diff
@@ -1513,6 +1514,25 @@ const Call = SparkPlugin.extend({
...
     });
+  },
+
+  /**
+   * Returns a {@link Readable} that emits {@link Call#media.pc}'s
+   * {@link RTCStatsReport} every second.
+   * @returns {StatsStream}
+   */
+  getRawStatsStream() {
+    // TODO stop polling after hangup
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/plugin-phone/src/call.js]: This TODO will be in another PR?

File: packages/node_modules/samples/browser-single-party-call-with-mute/test/wdio/spec/normal-dialing.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      assert.equal(browserSpock.getText('#outgoing-video-stats'), noStreamText);
+      browserSpock.click('[title="start sending video"]');
+      waitForSpecificText(browserSpock, '#outgoing-video-stats', noStreamText, true);
+    });
+
+    it('ends the call', () => {
+      browser.pause(5000);
+      // TODO add assertions around streams
+
+      browserSpock.click('[title="hangup"]');
```
Comment (adamweeks) [inline packages/node_modules/samples/browser-single-party-call-with-mute/test/wdio/spec/normal-dialing.js]: This test doesn't test anything.

File: packages/node_modules/@ciscospark/plugin-phone/src/call.js (hunk 1 above)
Comment (ianwremmel) [reply packages/node_modules/@ciscospark/plugin-phone/src/call.js]: actually, it's already done, i just need to remove the TODO

File: packages/node_modules/samples/browser-single-party-call-with-mute/test/wdio/spec/normal-dialing.js (hunk 2 above)
Comment (ianwremmel) [reply packages/node_modules/samples/browser-single-party-call-with-mute/test/wdio/spec/normal-dialing.js]: yea, none of these guys really do yet. I'm going to tackle that in the PR i'm starting right now (unifying sample code, creating more informative test helpers, etc).

**Comment by ianwremmel** [comment]
merged via #795
